rero_ils/modules/acq_orders/jsonschemas/acq_orders/acq_order-v0.0.1.json

```diff
@@ -0,0 +1,235 @@
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {
+              "term": "library_order_number"
+            }
+          },
+          "messages": {
+            "alreadyTakenMessage": "The Library order number is already taken"
+          }
+        }
+      }
+    },
+    "order_type": {
+      "title": "Order type",
+      "type": "string",
+      "enum": [
+        "serial",
+        "monograph",
+        "standing order",
+        "monographic set",
+        "planned order",
+        "multi-volume"
+      ],
+      "default": "monograph",
+      "form": {
+        "options": [
+          {
+            "value": "monograph",
+            "label": "Monograph"
+          },
+          {
+            "value": "serial",
+            "label": "Serial"
+          },
+          {
+            "value": "standing order",
+            "label": "Standing order"
+          },
+          {
+            "value": "monographic set",
+            "label": "Monographic set"
+          },
+          {
+            "value": "planned order",
+            "label": "Planned order"
+          },
+          {
+            "value": "multi-volume",
+            "label": "Multi-volume"
+          }
+        ]
+      }
+    },
+    "description": {
+      "title": "Description",
+      "type": "string",
+      "minLength": 5
+    },
+    "order_status": {
+      "title": "Order status",
+      "type": "string",
+      "enum": [
+        "approved",
+        "canceled",
+        "ordered",
+        "requested",
+        "pending",
+        "received"
+      ],
+      "default": "pending",
+      "form": {
+        "options": [
+          {
+            "value": "approved",
+            "label": "Approved"
+          },
+          {
+            "value": "canceled",
+            "label": "Canceled"
+          },
+          {
+            "value": "ordered",
+            "label": "Ordered"
+          },
+          {
+            "value": "requested",
+            "label": "Requested"
+          },
+          {
+            "value": "pending",
+            "label": "Pending"
+          },
+          {
+            "value": "received",
+            "label": "Received"
+          }
+        ]
+      }
+    },
+    "currency": {
+      "title": "Order currency",
+      "type": "string",
+      "enum": [
+        "CHF",
+        "EUR",
+        "USD"
+      ],
+      "default": "CHF"
+    },
+    "order_date": {
+      "title": "Order date",
+      "type": "string",
+      "format": "date",
+      "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+      "validationMessage": "Required. Should be in the ISO 8601, YYYY-MM-DD.",
+      "form": {
+        "type": "datepicker",
+        "placeholder": "Choose a date",
+        "wrappers": [
+          "form-field"
+        ],
+        "validation": {
+          "messages": {
+            "pattern": "Required. Should be in the ISO 8601, YYYY-MM-DD."
```

Comment:
this is a typical message for developer. Can you replace "YYYY-MM-DD" by and example?

Reply:
done